packages/calling/src/Voicemail/BroadworksBackendConnector.ts

```diff
@@ -465,7 +465,9 @@ export class BroadworksBackendConnector implements IBroadworksCallBackendConnect
       return responseDetails;
     } catch (err: unknown) {
       /* Catch the exception error code from try block, return the error object to user */
-      const errorInfo = err as WebexRequestPayload;
+      const errorInfo = {
+        statusCode: err instanceof Error ? Number(err.message) : '',
```

Comment:
Can we not set '' as default statusCode. Put 422 or equivalent code.

```suggestion
        statusCode: err instanceof Error ? Number(err.message) : 422,
```

Reply:
We're setting default statusCode as 422 in `serviceErrorCodeHandler` in the default switch statement when it doesn't matches with any of the status code. If we want to set it default here then we have to update it at multiple places wherever it is used. Do you still suggest changing this at multiple places or current implementation is acceptable?

Reply:
Thats fine . Ignore this if it was like this